### PR TITLE
Feature/add opinion

### PR DESCRIPTION
**Pull request by michaszcz**
Implement #46

### CODE REVIEW

File: website/views.py [hunk 1]
```diff
@@ -346,3 +349,71 @@ def post(self, request, pk):
...
+        business = get_object_or_404(Business, pk=pk)
+
+        return render(request, self.template_name, {
+            'add_opinion_form': self.form(None),
+            'business': business
+        })
+
+    def post(self, request, pk):
+        if not request.user.is_client():
+            raise Http404
```
Comment (fisheye36) [inline website/views.py:371]: `Http404()`

File: website/templates/website/pages/add_opinion.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                {{ field }}
+            </div>
+        {% endfor %}
+    </div>
+    <div class="footer text-center">
+        <button type="submit" class="btn btn-primary btn-round btn-lg btn-block">Add Opinion!</button>
+    </div>
+</form>
+
+{% endblock %}
```
Comment (fisheye36) [inline website/templates/website/pages/add_opinion.html]: No newline at the end of the file.

**Review by fisheye36 — CHANGES_REQUESTED** [review]
Fix commented lines.

**Review by ptr97 — APPROVED** [review]

File: website/templates/website/pages/add_opinion.html (hunk 2 above)
Comment (michaszcz) [reply website/templates/website/pages/add_opinion.html]: This is only temporary template to pass view tests.

**Review by fisheye36 — APPROVED** [review]

**Pull request merged**